app/tasks/maintenance.php

```diff
@@ -99,7 +99,7 @@ function notifyDeleteExpiredSessions()
         {
             (new Delete())
                 ->setType(DELETE_TYPE_SESSIONS)
-                ->setDatetime(DateTime::addSeconds(new \DateTime(), -1 * Auth::TOKEN_EXPIRATION_LOGIN_LONG))
+                ->setDatetime(DateTime::addSeconds(new \DateTime(), -1 * Auth::TOKEN_EXPIRATION_LOGIN_LONG)) //TODO: Update to use project session expiration instead of default.
```

Comment:
Reminder.

Reply:
I'll probably have to remove this comment as I don't think it's possible to access the project document from the maintenance task

Reply:
In theory, I don't even think we need this timestamp. We can just delete all expired session in the deletes worker according to current timestamp + a safety factor of a few days and remove this var altogether. WDYT?

Reply:
how do we know they are expired if the session length is variable?